source/modules/MEI.stringtab.xml

```diff
@@ -104,9 +181,92 @@
-        The fret at which the barre should be created is recorded by the <att>fret</att>
+          scheme="MEI">chordMember</gi> elements on which the barre starts and finishes
+        respectively. The fret at which the barre should be created is recorded by the <att>fret</att>
         attribute.</p>
     </remarks>
   </elementSpec>
-</specGrp>
+  <elementSpec ident="course" module="MEI.stringtab" mode="add">
+    <gloss>individual course tuning information</gloss>
+    <desc xml:lang="en">Describes the tuning of a course on a stringed instrument (e.g., guitar, lute).</desc>
+    <classes>
+      <memberOf key="att.course.log"/>
+    </classes>
+    <content>
+      <rng:zeroOrMore>
+        <rng:ref name="string" />
+      </rng:zeroOrMore>
+    </content>
+  </elementSpec>
+  <elementSpec ident="tuning" module="MEI.stringtab" mode="add">
+    <gloss>instrument tuning information</gloss>
+    <desc xml:lang="en">Describes the tuning of an instrument.</desc>
+    <classes>
+      <memberOf key="att.tuning.log"/>
+    </classes>
+    <content>
+      <rng:zeroOrMore>
+        <rng:ref name="course" />
+      </rng:zeroOrMore>
+    </content>
+  </elementSpec>
+  <elementSpec ident="tabDurSym" module="MEI.stringtab">
+    <desc xml:lang="en">A visual indication of the duration of a <gi scheme="MEI">tabGrp</gi>.</desc>
+    <classes>
+      <memberOf key="att.common" />
+      <memberOf key="att.facsimile" />
+      <memberOf key="att.stringtab" />
+      <memberOf key="att.tabDurSym.log" />
+      <memberOf key="att.tabDurSym.vis" />
+      <memberOf key="att.tabDurSym.ges" />
+      <memberOf key="att.tabDurSym.anl" />
+    </classes>
+    <content>
+      <rng:empty />
+    </content>
+    <constraintSpec ident="check_tabDurSym_in_beams" scheme="isoschematron">
+      <constraint>
+        <sch:rule context="mei:tabDurSym">
+          <sch:assert role="warning" test="not(ancestor::mei:beam)">It is very uncommon to see tab
+            duration symbols inside beams.</sch:assert>
+        </sch:rule>
+      </constraint>
+      <!-- TODO Add constraint: note(s) and rest(s) should be mutually exclusive -->
+    </constraintSpec>
+    <remarks>
+      <p>The presence of a <gi scheme="MEI">tabDurSym</gi> indicates that there is a symbol rendered
+        for the parent <gi scheme="MEI">tabGrp</gi>. If no additional attributes like <att>altsym</att>are
+        given, a default symbol conforming to the current notation style or font is to be used. The
+        absence of this elements indicates that the last provided duration symbol equally applies to
+        the current tab group.</p>
+    </remarks>
+  </elementSpec>
+  <elementSpec ident="tabGrp" module="MEI.stringtab">
+    <desc xml:lang="en">A group of simultaneous tab notes, comparable to a <gi scheme="MEI">chord</gi>
+      in CMN. Rarely, may also contain rests, as in GLT (D-Mbs Mus. ms. 1512 fol. 4r).</desc>
```

Comment:
No idea what GLT is. It may be better to move this to the Guidelines, and cut out a picture from this MS as an example.

Reply:
German lute tablatures. Suggest we expand the acronym here, and then move the manuscript example to the guidelines with a picture as you've proposed

Reply:
We have now expanded the acronym and removed the specific example from this file. We will include it in the documentation in a future commit.